SpinCore_pp/ppg/echo.py

```diff
@@ -96,13 +96,14 @@ def run_spin_echo(
     tx_phases = r_[0.0, 90.0, 180.0, 270.0]
     nPhaseSteps = len(ph1_cyc) * len(ph2_cyc)
     data_length = 2 * nPoints * nEchoes * nPhaseSteps
+    for_RX_nScans = 1
     for nScans_idx in range(nScans):
         run_scans_time_list = [time.time()]
         run_scans_names = ["configure"]
         configureTX(adcOffset, carrierFreq_MHz, tx_phases, amplitude, nPoints)
         run_scans_time_list.append(time.time())
         run_scans_names.append("configure Rx")
-        acq_time_ms = configureRX(SW_kHz, nPoints, nScans, nEchoes, nPhaseSteps)
+        acq_time_ms = configureRX(SW_kHz, nPoints, for_RX_nScans, nEchoes, nPhaseSteps)
```

Comment:
I'm pretty sure configureRX is where we want to feed one scan but we might need to remove the `acq_time_ms = `  since this value is used in the actual ppg (line 121) and will overwrite the acq_time we want to use?

Reply:
Yes!